frontend/src/components/DocumentUnitList.vue

```diff
@@ -196,12 +197,26 @@ function onDelete() {
             listEntry.documentType ? listEntry.documentType.jurisShortcut : "-"
           }}
         </CellItem>
-        <CellItem class="min-w-176">
+        <CellItem class="flex min-w-176 flex-row">
           <IconBadge
             v-if="listEntry.status?.publicationStatus"
+            class="inline-flex"
             v-bind="useStatusBadge(listEntry.status).value"
             data-testid="publication-status"
           />
+          <span
+            v-if="
+              listEntry.status?.publicationStatus ===
+              PublicationState.EXTERNAL_HANDOVER_PENDING
+            "
+          >
+            {{
+              " aus " +
+              listEntry.source +
+              " von " +
+              listEntry.creatingDocumentationOffice
+            }}
```

Comment:
```suggestion
            {{
              " aus " +
              listEntry.source +
              " (" +
              listEntry.creatingDocumentationOffice +
              ")"
            }}
```

Reply:
Ich würde das Dokoffice aus Platzgründen wie in Viviens Design in Klammern dahinterschreiben

Reply:
Danke!